scripts/exregional_run_enkf.sh

```diff
@@ -160,6 +160,60 @@ mkdir_vrfy -p ${enkfanal_nwges_dir}
 #
 #-----------------------------------------------------------------------
 #
+# Loop through the members, copy over the background and
+#  observer output (diag*ges*) files to the running directory
+#
+#-----------------------------------------------------------------------
+#
+ cp_vrfy ${fixgriddir}/fv3_coupler.res    coupler.res
+ cp_vrfy ${fixgriddir}/fv3_akbk           fv3sar_tile1_akbk.nc
+ cp_vrfy ${fixgriddir}/fv3_grid_spec      fv3sar_tile1_grid_spec.nc
+
+#
+#-----------------------------------------------------------------------
+#
+# Get nlons (NX_RES) and nlats (NY_RES) from  fv3_grid_spec
+#
+#-----------------------------------------------------------------------
```

Comment:
Line 172-177: Are these comments correct here? I did not see any script snippet to "get nlons and nlats"

Reply:
Just cleaned the script. Thanks for catching this!